packages/web-components/fast-foundation/src/di/di.ts

```diff
@@ -1624,10 +1973,9 @@ function buildAllResponse(
 
 const defaultFriendlyName = "(anonymous)";
 
-/**
- * @alpha
- */
-export interface DOMParentLocatorEventDetail {
+// making this private because I think we may want to
+// refactor to use the new proposed community context standard
```

Comment:
Is there a link for the community context standard?

Reply:
I followed up with an rfc here: https://github.com/microsoft/fast/issues/4948 That contains the link to the community standard doc.